src/pages/Team.js

```diff
@@ -1,5 +1,25 @@
+import { Box, Grid } from "@material-ui/core/";
+// import Box from "@material-ui/core/Box";
+import TeamMemberList from "../components/TeamMemberList";
+
+import useMyStyles from "../styles/MyStyles";
+
 const Team = () => {
-  return <div>Hello Team</div>;
+  const classes = useMyStyles();
+
+  return (
+    <Grid
+      container
+      justify="center"
+      alignItems="center"
+      className={classes.grid__Page}
```

Comment:
try change classes in more generic way

Reply:
I removed this styling and styled inline using Material-UI props.